### PR TITLE
Me component (fixes #976)

### LINKED ISSUE #976: Create component for user account page
__Description:__

Create in Angular component for user's account view to hold all components related with current user account.

### PR DESCRIPTION
**Pull request by OtisRed**
@juliaanholcer is this what you had in mind in #976 story?

### CODE REVIEW

File: frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -130,10 +131,14 @@ const appRoutes: Routes = [
     component: PasswordResetComponent,
     canActivate: [LoggedOutGuard],
   },
+  {
+    path: 'me',
+    component: MeComponent,
+  },
```
Comment (juliaanholcer) [inline frontend/src/app/app.module.ts]: I think you can use guard here, to block access to this site for an anonymous user.

File: frontend/src/app/me/me.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+Tutaj znajdzie się widok użytkownika
```
Comment (juliaanholcer) [inline frontend/src/app/me/me.component.html]: Maybe you can plan also the more detailed layout of this view, e.g. put two ` div `s, one for personal data form and the second for the list of organizations, with proper bootstrap classes - placeholders to just put those two components when they will be ready.

File: frontend/src/app/me/me.component.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { Component } from "@angular/core";
```
Comment (juliaanholcer) [inline frontend/src/app/me/me.component.ts]: Change `"` to `'` in import, linter isn't happy with that ;-)

**Comment by magul** [comment]
From Travis build:
```
ERROR: /home/travis/build/CodeForPoznan/volontulo/frontend/src/app/me/me.component.ts[1, 27]: " should be '
ERROR: /home/travis/build/CodeForPoznan/volontulo/frontend/src/app/app.module.ts[48, 28]: " should be '
Lint errors found in the listed files.
The command "ng lint" exited with 2.
```

**Comment by magul** [comment]
BTW can we use `acount` name for this component. I'm aware, that an url here is `/me` but it feels strage to have this component named that way.

**Comment by magul** [comment]
Even original issue (#976 ) uses `user account page` in its title.

**Comment by OtisRed** [comment]
@magul You may not remember this but actually "/me" path was your idea couple of months ago when I was trying to take this task for the first time :D

File: frontend/src/app/account/account.component.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<div class="container">
+  <div class="row">
+    <div class="col-6">
+      <div class="card">
+        <h3 class="card-header">Pictures</h3>
```
Comment (magul) [inline frontend/src/app/account/account.component.html]: Two things here - it's in English and after all we abandoned (at least for the time being) user profile pictures - see: #993

**Pull request merged**